src/UniGetUI/Controls/OperationWidgets/PackageOperations.cs

```diff
@@ -356,6 +358,31 @@ protected override async Task<AfterFinshAction> HandleSuccess()
                 await Package.AddToIgnoredUpdatesAsync(Package.NewVersion);
             }
 
+            if (Settings.Get("AskToDeleteNewDesktopShortcuts"))
```

Comment:
I don't remember if I forgot or just didn't but I'd like this to also be there for install operations as well (perhaps behind a setting?) Shouldn't be that hard and maybe I can even move this code up to the `PackageOperation` class.

Reply:
I will rebuild the PackageOperation class from scratch, so don't bother too much about it.

Reply:
And yes, it could be a good idea to have the feature to be enabled in different levels (disabled, on update only and on install and update)

Reply:
> I will rebuild the PackageOperation class from scratch, so don't bother too much about it.

What do you mean rebuild it from scratch? Right now or during a refactor later? And is there any of this (specifically the setting) you'd like me to do?

Reply:
I want to do it as soon as I can, since I want to fully detach the UI from the internal engine. The final goal is that MainWindow object can be destroyed with UniGetUI fully running on the background. (so much less resources are used on the background)

Also, I want to include a new operation to download installers, because the current download method is quite sketchy.


> And is there any of this (specifically the setting) you'd like me to do?

Don't worry about it, I am still thinking how to make the changes. I'll take care of this